src/ihela_client/banking_client.py

```
@@ -0,0 +1,412 @@
+    ) -> dict[str, Any]:
+        url = BANKING_ENDPOINTS["DEPOSIT"]
+        payload = {
+            "credit_account": credit_account,
+            "credit_account_holder": credit_account_holder,
+            "amount": amount,
+            "description": description,
+            "external_reference": external_reference,
+            "pin_code": pin_code,
+            "external_code": external_code,
+        }
+        resp = requests.post(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)